### ISSUE TITLE
Hobbed bolt not center to input filament 

### ISSUE THREAD

**Issue by alanNz** [firepick-delta/firepick-delta#18]
For some reason it was way off on mine, I reduce the spacer 3DAT0028 by 3 - 3.5 mm to get alignment

**Comment by FlyingLotus1983** [comment]
I designed the spacer specifically for the RobotDigg hobbed bolt's spacing.  On my machines here, they match up perfectly. 

I would like to see some measurements from your hobbed bolt to see if it's simply manufacturing variation from the hobbing process, or if it's something else.  The measurement I'm wondering about is from the bottom of the bolt head to the center of the hobbed area.  Could you measure that?  Also, could you post pics of your setup, both assembled and un-assembled?  

And is it possible that there's some support material in the bearing pocket of the extruder housing?  That would cause a mis-alignment.  That area needs to be perfectly smooth, and might take a razor blade or other tool to get it perfect.

**Comment by alanNz** [comment]
the bearing in the plastic gear side sits flush, the hex head of the bolt sits flush, the pocket looked pretty clean. the Hobbed bolt measurement is 27mm to the center
![img_20150607_101834](https://cloud.githubusercontent.com/assets/5855447/8021779/33b656a6-0d02-11e5-8d45-595ca77a74e5.jpg)
![img_20150607_103251](https://cloud.githubusercontent.com/assets/5855447/8021780/35563a26-0d02-11e5-8d6d-137d67ce8a8f.jpg)
![img_20150607_103644](https://cloud.githubusercontent.com/assets/5855447/8021781/3731b37a-0d02-11e5-8699-04977afcfc29.jpg)

**Comment by FlyingLotus1983** [comment]
I think I see the issue now :(

Your spacer should be 8.5mm.  I'm wondering if this was a failed part that should have been thrown away.  Could you send pictures of both top and bottom side?  If this is in fact the problem, I can mail you out a replacement spacer.

EDIT: Never mind, I Just noticed you trimmed it down to make it fit.

The hobbed distance should be at 30mm by the way, so 27 is probably close enough.

**Comment by alanNz** [comment]
I've reduced the size of that spacer until it lined up so it probably was a good print. 3mm difference in center is huge! well to me it is lol 

I didn't take a before shot unfortunately, it looked like the hobbed bolt would slip/ jam on the filament as it came thru since its going across the smooth section. I've re created what the 3 mm offset looked like in the attached picture.

From a kit point of view you could pack 10 x 1mmwashers so it can be tweaked to overcome manufacturing defects.
![img_20150607_111039](https://cloud.githubusercontent.com/assets/5855447/8021894/92efbf0a-0d06-11e5-8ccf-15df38566869.jpg)

**Comment by FlyingLotus1983** [comment]
OK, I understand now.  I think your hobbed bolt is the problem then.  Sounds like you've made it work, so that's good.  We'll certainly try to come up with a more fool-proof way to do this in the future.  I personally measured 10-15 of them before I made that spacer.  I felt that was better than 4-7 standard M8 washers, which have a pretty significant tolerance stack-up.

**Comment by alanNz** [comment]
yea washer's don't look pro either. If you made a 5.5mm plastic spacer + 2mm spacer + 1mm spacer + .5 spacer etc. or 5.5mm, 6.5mm 7.5mm, 8.5mm  etc. Or turn the hobbed bolt into a hobbed threaded rod then you have your double nut on both ends for infinite adjustment.

**Comment by alanNz** [comment]
or you just add the step i did, rub spacer on sand paper until it lines up. And then get you hobbed bolt guy to get there act together and spec to +/- 0.5mm which "should" be easy on these!

**Comment by FlyingLotus1983** [comment]
Right.  E3D sells a hobbed bolt with threaded ends on both sides, they did this for the same reason.. it makes it a lot more adjustable.  Sadly their price is probably 10x to 25x the price of the one we used in this kit.

**Comment by alanNz** [comment]
I always liked the look of these, used in diferent extruder that I've never actually used so cant comment on how well they actually work http://www.aliexpress.com/item/J35-Free-Shipping-Extruder-MK7-Stainless-Steel-Drive-Gear-Hobbed-Gear-For-Reprap-3D-Printer/32219482655.html

**Comment by FlyingLotus1983** [comment]
Several problems with those (direct drive) types.  Lower torque, and heat conducted from the stepper motor into the hobbed disc can cause the filament to go mushy.  Lots of people use them, but I've never been a fan of them. I do really like Wade extruders, but as you've seen, they can be trick to setup.  The good news is that after they're setup, they require little to no maintenance, and have proven to be very reliable.

**Comment by cyberreefguru** [comment]
I had to shave my spacer down to ~5.8mm to get the hobbed bolt to align properly.  YMMV.
